CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ BUG FIXES:
 IMPROVEMENTS:
 * Helm
   * API Gateway: Allow controller to read ReferencePolicy in order to determine if route is allowed for backend in different namespace. [[GH-1148](https://github.com/hashicorp/consul-k8s/pull/1148)]
+  * API Gateway: Re-use connectInject.consulNamespaces instead of requiring apiGateway.consulNamespaces have the same value. [[GH-1169](https://github.com/hashicorp/consul-k8s/pull/1169)]
```

Comment:
It should be a breaking change I think?

Reply:
I agree it _feels_ like a breaking change, but I'm still thinking on it.My understanding is that there's no valid case where `apiGateway.consulNamespaces` differs from `connectInject.consulNamespaces`, and it doesn't break if you do something like `helm install ... --set apiGateway.consulNamespaces...` even after removing from `values.yaml`.

Reply:
Overall I think better safe than sorry. I'm sure there's a contrived case where this would be a breaking change. For example, I've got `gateway.destNamespace=default` and I've got my gateway in kube ns `foo` and I've got `inject.mirroring=true`. Now with this change my gateway will move from `default` to `foo`.

Reply:
Good point! Addressed